### ISSUE TITLE
Upgrade from 1.0.0-beta5 to 1.0.0 breaks my webpack build

### ISSUE THREAD

**Issue by larrydahooster** [ericgio/react-bootstrap-typeahead#140]
Since the upgrade to 1.0.0 my project does not build anymore and I am getting this error output:

```
ERROR in ./~/react-bootstrap-typeahead/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./AsyncTypeahead in /[...]/node_modules/react-bootstrap-typeahead/lib
 @ ./~/react-bootstrap-typeahead/lib/index.js 8:23-50

ERROR in ./~/react-bootstrap-typeahead/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./Menu in /[...]/node_modules/react-bootstrap-typeahead/lib
 @ ./~/react-bootstrap-typeahead/lib/index.js 12:13-30

ERROR in ./~/react-bootstrap-typeahead/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./MenuItem in /[...]/node_modules/react-bootstrap-typeahead/lib
 @ ./~/react-bootstrap-typeahead/lib/index.js 16:17-38

ERROR in ./~/react-bootstrap-typeahead/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./Token in /[...]/node_modules/react-bootstrap-typeahead/lib
 @ ./~/react-bootstrap-typeahead/lib/index.js 20:14-32

ERROR in ./~/react-bootstrap-typeahead/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./Typeahead in /[...]/node_modules/react-bootstrap-typeahead/lib
 @ ./~/react-bootstrap-typeahead/lib/index.js 24:18-40
```

As far as I see you changed the imports in the index.js to

```
export AsyncTypeahead from './AsyncTypeahead';
export Menu from './Menu';
export MenuItem from './MenuItem';
export Token from './Token';
export Typeahead from './Typeahead';
```

from 

```
export AsyncTypeahead from './AsyncTypeahead.react';
export Menu from './Menu.react';
export MenuItem from './MenuItem.react';
export Token from './Token.react';
export Typeahead from './Typeahead.react';
```

but still the files in the project are named [Component].react.js.

**Comment by ericgio** [comment]
Goddammit. Thanks for the report.

**Comment by ericgio** [comment]
Note that as a workaround, you should be able to add `.react.js` to `resolve.extensions` in your `webpack.config`:

```
resolve: {
  extensions: ['', '.js', '.json', '.jsx', '.react.js']
}
```

**Comment by ericgio** [comment]
Fixed in v1.0.2

SORRY

**Comment by larrydahooster** [comment]
Hey no problem :) Thanks for the quick fix and all the work you've put into this library. Pretty nice component!
